README.md

```diff
@@ -321,3 +321,21 @@ contract = [
 ```
 
 Validate that the supplied value is contained within the argument whitelist.
+
+### At least one required
+
+```javascript
+contract = [
+  {
+    key: 'age', // ingored in this case
```

Comment:
might be worth making both these fields optional, and then validating on it later?

Reply:
actually, looking at code, could we just use the existing `key` field, and make a check for it being an array?

Reply:
I thought that - but we cant do that easily  as key can already be an array - defining a path to a nested property in the data

Reply:
oh of course, hadn't actually though about that, looks good to me then. but I think it makes sense to make the key prop optional, to remove potentially redundant data